salesagility/SuiteCRM
# Feature acceptance tests - Install Test Suite

## Description



This change adds a new acceptance test suite. I have created some helper classes which can be reused during acceptance testing.

## Motivation and Context


## How To Test This

* Install docker compose
* Use the selenium hub image with chrome image
* See tests/readme.md

## Types of changes


### Final checklist
